packages/core/src/utils.js

```diff
@@ -140,7 +140,8 @@ module.exports = {
         const ext = opts.ext || '';
 
         if (toPath.startsWith('.')) {
-            return ext === '' ? toPath : toPath + ext;
+            const toPathExt = /^.+\.([^.]+)$/.exec(toPath);
```

Comment:
Use `Path.extname(toPath)` as is used a few lines below - `Path.extname(toPath) ? toPath : toPath + ext`.

Reply:
D'oh, good suggestion!

Reply:
I have mode this update.